app/views/fact_values/index.html.erb

```diff
@@ -13,7 +13,10 @@ if params[:host_id].present?
       {
         caption: _('values')
       }
-    ]
+    ],
+    resource_url: api_hosts_path(thin: true),
+    switcher_item_url: "/hosts/:name/facts",
```

Comment:
Couldn't you use a ruby helper here?

Reply:
`host_facts_path(params[:host_id])`

Reply:
```host_facts_path(':name')``` should give you the same result i believe in a way that won't break if this path ever changes.

Reply:
Thanks, works